### DISCUSSION TITLE
Wallet address automatic changed unable receive XCH

### DISCUSSION THREAD

**Discussion by kenyipchoo** [Chia-Network/chia-blockchain#9287] [Support]
Dear All,

Yesterday I had transferred XCH to another gate.io wallet, but until now haven't received it. Unfortunately I found out the wallet address had changed without any notification.

https://xchscan.com/txns/0x14525b5b98681e95355c93a76a5ec0808e9399a57645d7172e3f25aa2b3f5357

0x14525b5b98681e95355c93a76a5ec0808e9399a57645d7172e3f25aa2b3f5357

Old Wallet Address
xch1wled5cktm47838z49wlmldp4aa9cflar874wrx3xgu2drx4tfhssxzh2jk

New Wallet Address
xch1nd6tp4j9x6zyus9a7gfzalma7p7dcz8tzh7m2mks7ze7wksv9fqqwkqv9k

Anyone can guide me how to link back to my Old wallet address? Please help.

**Comment by OldButtKicking** [comment]
This should not be a problem (I realize it does appear to be) you can have many addresses for the same wallet. xch sent to the old address should still end up in your wallet if this is the wallet associated with that address. Can you confirm the transaction on a block chain explorer and this is not by chance a new wallet?
?
